ocraft-s2client-protocol/src/main/java/com/github/ocraft/s2client/protocol/data/Upgrades.java

```diff
@@ -380,21 +384,6 @@ public static Upgrade from(int sc2ApiUpgradeId) {
         return Optional.ofNullable(upgradeIdMap.get(sc2ApiUpgradeId)).orElse(Other.of(sc2ApiUpgradeId));
     }
 
-    public static void remapForBuild(Integer build) {
```

Comment:
The purpose of this method is to adapt the id to the build with which the API has connected, it should not be removed, but extended to subsequent builds.

Reply:
There was this brief period of time where Blizzard changed a bunch of ids.  This is when I added that Upgrades.remapForBuild() method: https://github.com/ocraft/ocraft-s2client/pull/71/files

About a month or two later, Blizzard changed these ids back, for the same patch number.  That is why this method is no longer needed, as there is no longer a conflict with id:296 between patches.